examples/contracts/examples/deploy_from_artifact.rs

```diff
@@ -50,9 +50,10 @@ async fn main() -> Result<()> {
     println!("Incremented number: {}", tx_hash);
 
     // Retrieve the number, which should be 43.
-    let Counter::numberReturn { _0 } = contract.number().call().await?;
+    let builder = contract.number();
+    let number = builder.call().await?._0;
 
-    println!("Retrieved number: {_0}");
+    println!("Retrieved number: {number}");
```

Comment:
@DaniPopes there appears to be a bug here, I am unable to access `.number` like I can with the other examples (deploy_from_bytecode, deploy_from_contract)

Reply:
I'll report it in `alloy-core` if I can narrow it down